<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Motor Stand</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>700</width>
      <height>250</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>250</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>700</width>
      <height>250</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="deviceTable">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>461</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="startTestBtn">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>210</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Запись</string>
     </property>
     <property name="icon">
      <iconset resource="Resources/Resources.qrc">
       <normaloff>:/images/images/start.png</normaloff>:/images/images/start.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="stopTestBtn">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>210</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Стоп</string>
     </property>
     <property name="icon">
      <iconset resource="Resources/Resources.qrc">
       <normaloff>:/images/images/stop.png</normaloff>:/images/images/stop.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="openPlotBtn">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>170</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Показать графики</string>
     </property>
     <property name="icon">
      <iconset resource="Resources/Resources.qrc">
       <normaloff>:/images/images/wave-graph.png</normaloff>:/images/images/wave-graph.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="plotMenu">
    <property name="title">
     <string>Графики</string>
    </property>
    <addaction name="openPlotAction"/>
    <addaction name="globalPlotSettingsAction"/>
   </widget>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="saveAllAction_2"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="plotMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openPlotAction">
   <property name="icon">
    <iconset resource="Resources/Resources.qrc">
     <normaloff>:/images/images/wave-graph.png</normaloff>:/images/images/wave-graph.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="saveAllAction_2">
   <property name="text">
    <string>Сохранить всё</string>
   </property>
  </action>
  <action name="globalPlotSettingsAction">
   <property name="icon">
    <iconset resource="Resources/Resources.qrc">
     <normaloff>:/images/images/settings.png</normaloff>:/images/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>Параметры</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
